src/utilities/filters-utils.js

```diff
@@ -0,0 +1,34 @@
+const getSortedProducts = (products, sortBy) => {
+  if (sortBy === "HIGH_TO_LOW") {
+    return [...products].sort((a, b) => b.discoutPrice - a.discoutPrice);
+  }
+  if (sortBy === "LOW_TO_HIGH") {
+    return [...products].sort((a, b) => a.discoutPrice - b.discoutPrice);
+  }
+  return products;
+};
+
+const getFilteredProducts = (
+  products,
+  stockBoolean,
+  deliveryBoolean,
+  categoryObj,
+  brands,
+  rating,
+  range
+) => {
+  console.log(rating);
+  return products
+    .filter((product) => (stockBoolean ? true : product.inStock))
```

Comment:
try to give good variable name like instead stockboolean you can name it itemInStock